woocommerce/woocommerce
# Set mailpoet 70%, klaviyo  30%

### Submission Review Guidelines:

Warning: This PR should go out on Jan 2nd.

Slack thread: p1734716737990699-slack-C01SFMVEYAK

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

- Set Mailpoet 70% and Klaviyo 30%




### How to test the changes in this Pull Request:

1. Create a new JN site.
3. Do not start the core profiler yet.
4. Go to WooCommerce -> Settings -> Advanced -> WooCommerce.com and uncheck `Show Suggestions` option.
6. Go to `/wp-admin/options.php`
7. Look for `woocommerce_remote_variant_assignment` and set it to `84` and save by pressing enter
5. Start the core profiler and proceed to the extensions page.
10. Observe Mailpoet is displayed
11. Set `woocommerce_remote_variant_assignment` to `85`
12. Refresh extensions step and observe Klaviyo is displayed
13. Ensure learn more link points to Klaviyo extension page



### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
